ibm/resource_ibm_kms_key_alias.go

```diff
@@ -0,0 +1,248 @@
+package ibm
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"net/url"
+	"strings"
+
+	kp "github.com/IBM/keyprotect-go-client"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+)
+
+func resourceIBMKmskeyAlias() *schema.Resource {
+	return &schema.Resource{
+		Create:   resourceIBMKmsKeyAliasCreate,
+		Delete:   resourceIBMKmsKeyAliasDelete,
+		Read:     resourceIBMKmsKeyAliasRead,
+		Importer: &schema.ResourceImporter{},
+
+		Schema: map[string]*schema.Schema{
+			"instance_id": {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "Key ID",
+				ForceNew:    true,
+			},
+			"alias_name": {
```

Comment:
Do we have alias_name and alias_id bith from API..if not just rename this as alias